depthai_ros_driver/include/depthai_ros_driver/ros_agnostic.hpp

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (C) 2021 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+#ifndef DEPTHAI_ROS_DRIVER__ROS_AGNOSTIC_HPP
+#define DEPTHAI_ROS_DRIVER__ROS_AGNOSTIC_HPP
+
+#if defined(USE_ROS2)
+  #include <rclcpp/node.hpp>
+  #define ROS_LOGGER(...) RCUTILS_LOG_WARN_NAMED("depthai_ros", __VA_ARGS__)
+using ROSNodeHandle = std::shared_ptr<rclcpp::Node>;
```

Comment:
Is this indentation manually created or by clang-format?

Reply:
yes is a linting, specifically using [uncrustify](http://uncrustify.sourceforge.net/)